app/jobs/plexer_job.rb

```diff
@@ -11,6 +11,23 @@ class PlexerJob < ApplicationJob
   # @param [String] druid
   # @param [Integer] version
   def perform(druid, version)
-    S3EndpointDeliveryJob.perform_later(druid, version)
+    targets(druid, version).each do |worker|
+      worker.perform_later(druid, version)
+    end
+  end
+
+  # @return [Array<Class>] EndpointDeliveryBase-descending classes to be targeted
+  def targets(druid, version)
```

Comment:
shouldn't the preservation_policy be used somewhere here? (cause thats where we will know which endpoint to send the druid/version?

Reply:
talked F2F - the plexer worker should already know which endpoint to target based on the pres_copy rows being existing. The preservation_policy logic can be implemented else where (like if it is not `default` and it is only `s3-west`)

Reply:
i think the approach here is "any archive endpoint that has a preserved copy for the given druid/version combo", under the assumption that the preserved copies have already been created in `UNREPLICATED_STATUS` (according to the endpoints specified by the pres policy).  so... i think this is ok?

Reply:
yup